util/linux_install.sh

```diff
@@ -208,19 +208,13 @@ elif grep ID /etc/os-release | grep -q solus; then
 	printf "\n$SOLUS_INFO\n"
 
 elif grep ID /etc/os-release | grep -q void; then
-	# musl Void systems don't have glibc cross compilers avaliable in their repos.
-	# glibc Void systems do have musl cross compilers though, for some reason.
-	# So, default to musl, and switch to glibc if it is installed.
-	CROSS_ARM=cross-arm-linux-musleabi
-	if xbps-query glibc > /dev/null; then # Check is glibc if installed
-		CROSS_ARM=cross-arm-linux-gnueabi
-	fi
-
 	sudo xbps-install \
 		avr-binutils \
 		avr-gcc \
 		avr-libc \
-		$CROSS_ARM \
+		cross-arm-none-eabi-binutils \
+		cross-arm-none-eabi-gcc \
+		cross-arm-none-eabi-newlib \
```

Comment:
Is it a guarantee that QMK won't need C++? Otherwise it's probably easier to install `cross-arm-none-eabi`, the only package it installs in addition to these is `cross-arm-none-eabi-libstdc++`.

Reply:
Good question. I don't see any other places in this script where the other distros include libstdc++. Of all the .cpp files I could find in the repo, the vast majority of them were in the USB Host Shield and ugfx libraries, which are AVR-only, and the test suite, which does not run on actual hardware.

The remaining files are:
```
./lib/chibios/demos/STM32/RT-STM32F407-DISCOVERY-G++/main.cpp
./lib/chibios/os/various/cpp_wrappers/ch.cpp
./lib/chibios/os/various/cpp_wrappers/syscalls_cpp.cpp
./lib/chibios-contrib/os/various/memtest.cpp
./tmk_core/protocol/lufa/adafruit_ble.cpp
./tmk_core/protocol/lufa/ringbuffer.hpp
```
The ChibiOS stuff appears to be specific only to that `RT-STM32F407-DISCOVERY-G++` demo board, and isn't used anywhere else. As for adafruit_ble.cpp and ringbuffer.hpp, they're currently AVR-only and IMO should not be written in C++ if it can be helped. I plan on tidying up the Bluetooth support anyway to make it more platform-agnostic, so I'll cross that bridge when I come to it.